client/packages/system/src/Item/Components/PackVariant/PackVariantEntryCell.tsx

```diff
@@ -19,9 +19,11 @@ export const PackVariantEntryCell =
   <T extends RecordWithId>({
     getItemId,
     getUnitName,
+    getIsDisabled,
```

Comment:
I don't think we need getIsDisabled, I think isDisabled is already available, but in the CellProps (below). Since it was working previous for PositiveNumberInputCell.

Reply:
That isn't working because of how the components are returned in the PackVariantEntryCell...  https://github.com/msupply-foundation/open-msupply/pull/2533#discussion_r1402664023